docs/user/library/cql/ecql.rst

```diff
@@ -204,6 +210,14 @@ Examples
         
         Expression expr = ECQL.toExpression("X + 1");
 
+  Literal string::
+      
+        Expression expr = ECQL.toExpression("'hello world'");
+
+  Literal string with quote::
```

Comment:
might want to put the word "escape" or something here so people can easily search for it.